### PR TITLE
[Check-in] Add `client_secret` to Device Authorization Request

### PR DESCRIPTION
**Pull request by nikosev**
# Summary

This PR adds the `client_secret` parameter to the examples of the Device Authorization Requests.

### CODE REVIEW

File: content/en/providers/check-in/sp/_index.md [hunk 1]
```diff
@@ -1040,6 +1043,13 @@ the following HTTP response during the authentication flow:
 error=invalid_request&error_description=Missing parameter: code_challenge_method
 ```
 
+##### Device Code
+
+If you are using the Device Code grant, please make sure that the
```
Comment (NicolasLiampotis) [inline content/en/providers/check-in/sp/_index.md:1048]: If you are using a confidential client with the Device Code grant

**Review by vardizzo-lab — CHANGES_REQUESTED** [review]
This is the new EGI policies link: 
https://confluence.egi.eu/display/EGIPP 
Could you correct it please?

**Comment by gwarf** [comment]
> This is the new EGI policies link: https://confluence.egi.eu/display/EGIPP Could you correct it please?

https://go.egi.eu/policies-and-procedures is the link that is expected to be used, we would update it later in case it would be changed.
@nikosev please revert back to this one.

File: content/en/providers/check-in/sp/_index.md [hunk 2]
```diff
@@ -136,8 +136,8 @@ provide the following information to connect your service to EGI Check-in:
    - have a size larger than 40000 and smaller than 50000 characters when
      encoded in base64
 1. Country of the service
-1. Compliance with the
-   [EGI Policies](https://go.egi.eu/policies-and-procedures) and the
+1. Compliance with the [EGI Policies](https://go.egi.eu/policies-and-procedures)
+   and the
    [GÉANT Data Protection Code of Conduct](https://wiki.refeds.org/display/CODE/Data+Protection+Code+of+Conduct+Home)
```
Comment (gwarf) [inline content/en/providers/check-in/sp/_index.md:141]:
```suggestion
   [REFEDS Data Protection Code of Conduct](https://wiki.refeds.org/display/CODE/Data+Protection+Code+of+Conduct+Home)
```

**Review by gwarf — APPROVED** [review]
LGTM

**Comment by gwarf** [comment]
@vardizzo-lab is it fine with you to merge it like this? Any other comment?

**Review by vardizzo-lab — APPROVED** [review]

**Pull request merged**